education/blockchain/decentralization.md

```diff
@@ -1,11 +1,11 @@
 # Decentralization
-Most cryptocurrencies are decentralized, which means that nobody who owns cryptocurrency has more information or power than anyone else. 
+Most cryptocurrencies are decentralized, which means that nobody who owns any amount of cryptocurrency has more information or power than anyone else. 
 
 ### Think of it like this
 In a decentralized bank, every customer gets a copy of the bank records(**blockchain**), so everyone can see how much money everyone else has.
 
 ## Why is decentralization important?
 
-Decentralization spreads out the information, so no one user/person/entity has absolute control over it.  Normal banks are *centralized*; although we all put our money into it, the bank has the final say in how much each person has. If I put in $100 to my account, the only people that know how much I deposited are me and the bank. The bank could just tell every other customer that I only have $1 in my account, and there isn't much I can do to prove them wrong. In a decentralized bank, if a teller at the bank tries to tell everyone I only have $1, anyone can look at the bank records(**blockchain**) to see that I really have 100$.
+Decentralization spreads out the information, so no one user/person/entity has absolute control over it.  Normal banks are *centralized*; although we all put our money into it, the bank has the final say in how much each person has. If I put in $100 to my account, the only people that know how much I deposited are me, and the bank. The bank could just tell every other customer that I only have $1 in my account, and there isn't much I can do to prove them wrong. In a decentralized bank, if a teller at the bank tries to tell everyone I only have $1, anyone can look at the bank records(**blockchain**) to see that I really have 100$.
```

Comment:
Perhaps instead of talking about proving balance, we should talk about the security aspect?